CONTRIBUTING.md

```diff
@@ -29,4 +29,3 @@ Lastly, run the test command from the console:
 - `git tag -a vX.X.X -m 'vX.X.X'`
 - `git push --tags`
 - `npm publish`
-- Deploy packaged plugin in [mapbox-gl-js/plugins](https://github.com/mapbox/mapbox-gl-js/tree/mb-pages/plugins)
```

Comment:
My plan (🙏) is to eliminate the need for GL JS docs to know about plugin versions. No changes need to occur in the [leaflet plugin directory](http://leafletjs.com/plugins.html) per release of a downstream plugin. I hope to achieve this by simplifying the GL JS plugin directory, including removing some features. 

While that upstream PR is shaping up, I'll leave add a value line to these deployment docs

 > - Update the GL JS plugin directory to use this plugin's new version number